.circleci/config.yml

```diff
@@ -224,7 +224,7 @@ local_e2e_steps: &local_e2e_steps
     - <<: *exports
     # Install kind
     - run: |
-        curl -LO https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-Linux-amd64
+        curl -LO https://github.com/kubernetes-sigs/kind/releases/download/v0.10.0/kind-Linux-amd64
```

Comment:
I am not sure if we are hardcoding the cluster version elsewher, can you check? I am okay running tests with 1.20 (which is the default of kind v0.10) but we need to be aware of that.

Reply:
I've checked and I'm not aware of further usages of a raw cluster version. However, I'm changing it to `kind create cluster --image kindest/node:v1.20.2` so that we know which k8s version is being used regardless of the Kind version we decide to use.

Reply:
Just for consistency, since we're using the `v1.20.2` node, why not use the same version to `kubectl` above and DRY it up with a single `K8S_VERSION=v1.20.2` ?